src/protocols/multi_party_ecdsa/gg_2020/test.rs

```diff
@@ -367,55 +413,59 @@ fn keygen_orchestrator(
             return Err(err);
         }
         let (shared_keys, dlog_proof) = result_check.unwrap();
```

Comment:
instead of simple unwrap, this (and the same goes in other checks from other stages) I suggest to return Err() some verbose message if check fails (and take the value otherwise)